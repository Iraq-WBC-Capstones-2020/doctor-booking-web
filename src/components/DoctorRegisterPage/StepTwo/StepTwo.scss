.stepTwoFilds{
    width: 425px;
    label{
        font-style: normal;
        font-weight: 500;
        font-size: 18px;
        line-height: 24px;
        color: #000000;
    }

    .theSelector{
        height: 48px;
    }
    .form-control{
    margin-top: 5px;
    width: 425px;
    border: 2px solid #eaeaea;
    box-sizing: border-box;
    border-radius: 3px;
    font-size: 14px;
    }

}

