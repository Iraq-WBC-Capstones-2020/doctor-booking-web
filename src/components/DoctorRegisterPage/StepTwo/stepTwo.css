.stepTwoFilds {
  width: 425px;
}

.stepTwoFilds label {
  font-style: normal;
  font-weight: 500;
  font-size: 18px;
  line-height: 24px;
  color: #000000;
}

.stepTwoFilds .theSelector {
  height: 48px;
}

.stepTwoFilds .form-control {
  margin-top: 5px;
  width: 425px;
  border: 2px solid #eaeaea;
  -webkit-box-sizing: border-box;
          box-sizing: border-box;
  border-radius: 3px;
  font-size: 14px;
}
/*# sourceMappingURL=StepTwo.css.map */