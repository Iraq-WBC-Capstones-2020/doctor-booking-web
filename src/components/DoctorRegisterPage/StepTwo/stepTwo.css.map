{
    "version": 3,
    "mappings": "AAAA,AAAA,aAAa,CAAC;EACZ,KAAK,EAAE,KAAK;CAsBb;;AAvBD,AAGE,aAHW,CAGX,KAAK,CAAC;EACJ,UAAU,EAAE,MAAM;EAClB,WAAW,EAAE,GAAG;EAChB,SAAS,EAAE,IAAI;EACf,WAAW,EAAE,IAAI;EACjB,KAAK,EAAE,OAAO;CACf;;AATH,AAWE,aAXW,CAWX,YAAY,CAAC;EACX,MAAM,EAAE,IAAI;CACb;;AAbH,AAeE,aAfW,CAeX,aAAa,CAAC;EACZ,UAAU,EAAE,GAAG;EACf,KAAK,EAAE,KAAK;EACZ,MAAM,EAAE,iBAAiB;EACzB,UAAU,EAAE,UAAU;EACtB,aAAa,EAAE,GAAG;EAClB,SAAS,EAAE,IAAI;CAChB",
    "sources": [
        "StepTwo.scss"
    ],
    "names": [],
    "file": "StepTwo.css"
}