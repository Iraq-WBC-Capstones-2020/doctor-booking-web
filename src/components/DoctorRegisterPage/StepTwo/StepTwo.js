import React, { useState } from 'react';
import { Col, Form, Container, Row } from 'react-bootstrap';
import './stepTwo.css';

function StepTwo() {

    const specilty = ["Abdominal radiology",
         "Breast imaging",
         "Cardiothoracic radiology",
         "Cardiovascular radiology",
         "Chest radiology",
         "Emergency radiology",
         "Endovascular surgical neuroradiology",
         "Gastrointestinal radiology",
         "Genitourinary radiology",
         "Head and neck radiology",
         "Pulmonary disease",
            "Pulmonary disease", 
            "Sleep medicine",
            "Sports medicine",
            "Transplant hepatology"
        ];
    
return(

    
    <Container >
        <Form>
        <Row className="formRow">
            <Col lg={12}>
        
  <Form.Group className="stepTwoFilds" controlId="exampleForm.SelectCustom">
    <Form.Label>Speciality</Form.Label>
    <Form.Control className="theSelect" as="select" custom>
    {specilty.map(spec =>
      <option>{spec}</option>
     
    )}
    </Form.Control>
  </Form.Group>

</Col>
        </Row>
        <Row className="formRow">
        <Col lg={6}>
            <Form.Group className="stepTwoFilds">
    <label for="exampleFormControlTextarea1">Education</label>
    <textarea className="textArea form-control" id="exampleFormControlTextarea1" rows="3"></textarea>
              </Form.Group>
                   
            </Col>
            <Col lg={6}>
            <Form.Group className="stepTwoFilds">
    <label for="exampleFormControlTextarea1">Expeience</label>
    <textarea class="textArea form-control" id="exampleFormControlTextarea1" rows="3"></textarea>
              </Form.Group>
                   
            </Col>
        </Row>
        </Form>
    </Container>

)

}

export default StepTwo
