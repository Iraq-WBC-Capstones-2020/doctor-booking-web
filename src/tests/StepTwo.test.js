import React from 'react';
import { render } from '@testing-library/react';
import StepTwo from '../components/DoctorRegisterPage/StepTwo/StepTwo';
import renderer from 'react-test-renderer';
it('StepTwo render1', () => {
  render(<StepTwo />);
});

it('StepTwo renders correctly', () => {
  const tree = renderer.create(<StepTwo />).toJSON();
  expect(tree).toMatchSnapshot();
});
