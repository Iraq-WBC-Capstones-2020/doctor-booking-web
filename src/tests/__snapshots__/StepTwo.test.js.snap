// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StepTwo renders correctly 1`] = `
<div
  className="container"
>
  <form
    className=""
  >
    <div
      className="formRow row"
    >
      <div
        className="col-lg-12"
      >
        <div
          className="stepTwoFilds form-group"
        >
          <label
            className="form-label"
            htmlFor="exampleForm.SelectCustom"
          >
            Speciality
          </label>
          <select
            className="theSelect custom-select"
            id="exampleForm.SelectCustom"
          >
            <option>
              Abdominal radiology
            </option>
            <option>
              Breast imaging
            </option>
            <option>
              Cardiothoracic radiology
            </option>
            <option>
              Cardiovascular radiology
            </option>
            <option>
              Chest radiology
            </option>
            <option>
              Emergency radiology
            </option>
            <option>
              Endovascular surgical neuroradiology
            </option>
            <option>
              Gastrointestinal radiology
            </option>
            <option>
              Genitourinary radiology
            </option>
            <option>
              Head and neck radiology
            </option>
            <option>
              Pulmonary disease
            </option>
            <option>
              Pulmonary disease
            </option>
            <option>
              Sleep medicine
            </option>
            <option>
              Sports medicine
            </option>
            <option>
              Transplant hepatology
            </option>
          </select>
        </div>
      </div>
    </div>
    <div
      className="formRow row"
    >
      <div
        className="col-lg-6"
      >
        <div
          className="stepTwoFilds form-group"
        >
          <label
            htmlFor="exampleFormControlTextarea1"
          >
            Education
          </label>
          <textarea
            className="textArea form-control"
            id="exampleFormControlTextarea1"
            rows="3"
          />
        </div>
      </div>
      <div
        className="col-lg-6"
      >
        <div
          className="stepTwoFilds form-group"
        >
          <label
            htmlFor="exampleFormControlTextarea1"
          >
            Expeience
          </label>
          <textarea
            className="textArea form-control"
            id="exampleFormControlTextarea1"
            rows="3"
          />
        </div>
      </div>
    </div>
  </form>
</div>
`;
